### PR TITLE
rshim_usb.c: Make necessary changes for Mustang

### PR DESCRIPTION
**Pull request by asmaamellanox**
There are no differences in the registers.
The following are the changes made:
1) BF-3 product ID, this is just a place holder.
Still waiting for the HW folks to provide it.
2) BF-2 Vendor ID remains the same i.e. Mellanox
3) the channel is no longer supported in BF-3.
Instead, the register address is computed as:
(wValue << 16) + wIndex

### CODE REVIEW

File: src/rshim_usb.c [hunk 1]
```diff
@@ -75,23 +77,34 @@ static void rshim_usb_delete(rshim_backend_t *bd)
   free(dev);
 }
 
+static int chan_to_wValue(int chan, uint16_t ver_id)
+{
+  if (ver_id == RSHIM_BLUEFIELD_3)
+    return chan << 7;
+
+  return chan;
```
Comment (lsun100) [inline src/rshim_usb.c:85]: We might need a table to update both wValue & wIndex for BF3 (by setting the the wValue and adding an offset to wIndex ).

File: src/rshim_usb.c [hunk 2]
```diff
@@ -75,23 +77,34 @@ static void rshim_usb_delete(rshim_backend_t *bd)
   free(dev);
 }
 
+static int chan_to_wValue(int chan, uint16_t ver_id)
```
Comment (lsun100) [inline src/rshim_usb.c:80]: It's ok to use lower case as function name to be consistent.

**Review by lsun100 — COMMENTED** [review]
Could you also add "Signed-off-by:" just like other commits?

File: src/rshim_usb.c (hunk 2 above)
Comment (asmaamellanox) [reply src/rshim_usb.c]: done.

File: src/rshim_usb.c (hunk 1 above)
Comment (asmaamellanox) [reply src/rshim_usb.c]: done

**Comment by asmaamellanox** [comment]
done

**Review by lsun100 — APPROVED** [review]

**Pull request merged**